consumers/hidden-test-consumer/src/hidden_test_consumer/aws_client.py

```
@@ -0,0 +1,36 @@
+import os
+from threading import Lock
+from typing import Dict, Type
+import boto3
+
+AWS_ACCESS_KEY_ID = os.environ.get("AWS_ACCESS_KEY_ID")
+AWS_SECRET_ACCESS_KEY = os.environ.get("AWS_SECRET_ACCESS_KEY")
+AWS_REGION = os.environ.get("AWS_REGION")
+
+
+class AWSClient:
+    _instances: Dict[str, "AWSClient"] = (
```

Ruff (0.8.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)